pkgs/applications/misc/myanimelist-cli/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, curl
+, json_c
+, libbsd
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "myanimelist-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmakhack";
+    repo = "myanimelist-cli";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-EmdkPpYEUIk9hr6rbnixjvznKSEnTCSMZz/17BfHGCk=";
+  };
+
+  buildInputs = [
+    curl
+    json_c
+    libbsd
+  ];
+
+  # support LLVM and such
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace 'CC:=gcc' 'CC:=${stdenv.cc.targetPrefix}cc'
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -a bin $out
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Minimalistic command line interface for fetching user anime data from MyAnimeList";
+    longDescription = ''
+      Minimalistic command line interface for fetching user anime data from MyAnimeList.
+
+      You have to run this with the MYANIMELIST_CLIENT_ID environ variable set.
+      Where to get one: <https://myanimelist.net/apiconfig>.
+      Select the type `other`.
+    '';
+    homepage = "https://github.com/jmakhack/myanimelist-cli";
+    changelog = "https://github.com/jmakhack/myanimelist-cli/releases/tag/v${finalAttrs.version}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ pbsds ];
+    mainProgram = "mya";
+    platforms = platforms.all;
+    # FIXME: darwin maintainer needed
+    broken = stdenv.isDarwin; # src/mya.c:4:10: fatal error: 'argp.h' file not found
```

Comment:
Dear @NixOS/darwin-maintainers, can you help on this?

Reply:
@pbsds Have a look at this: https://github.com/NixOS/nixpkgs/pull/194312 it might probably be the solution.

Reply:
Yes, an `argp-standalone` build input is the way to go. Ideally it should be conditioned on "libc is not glibc" rather than Darwin specifcially (for instance, it applies to other BSDs), but I'm not sure if there's a convenient way to express this. If that isn't enough I can take another look.

Reply:
Maybe `lib.optionals (!stdenv.hostPlatform.isGnu)` is the right logic.